assets/shared-components/ctaLink.jsx

```diff
@@ -0,0 +1,24 @@
+// ----- Imports ----- //
```

Comment:
It's named that to make it explicit what it's doing, because there are going to be several directories called `components`. However, if you think it's clear enough I can rename it?